sdk/core/amqp-common/src/errors.ts

```diff
@@ -493,7 +493,8 @@ export const retryableErrors: string[] = [
   "MessagingError",
   "DetachForcedError",
   "ConnectionForcedError",
-  "TransferLimitExceededError"
+  "TransferLimitExceededError",
+  "OperationTimeoutError"
```

Comment:
Update the comment on this constant as well

Reply:
Updated.